### ISSUE TITLE
Resolves host forever - 1.10 development build

### ISSUE THREAD

**Issue by theonlygusti** [ORelio/Minecraft-Console-Client#213]
When I start MCC it outputs this:

```
Console Client for MC 1.4.6 to 1.10.2 - v1.10.0 DEV - By ORelio & Contributors
Resolving us.mineplex.com...
```

and then nothing else. It sits on that until I terminate it.

I have specified my username, password and the server in `MinecraftClient.ini`.

**Comment by ORelio** [comment]
See #208 ; if that happens for too many people I'll consider turning it off by default.

**Comment by theonlygusti** [comment]
I have now added this line to my `MinecraftClient.ini` file:

    resolvesrvrecords=false

but it has changed nothing.

**Comment by ORelio** [comment]
There should aleady be an entry in the file. If that is not the case make sure to add it to the **[Main]** section.

**Comment by ORelio** [comment]
Okay, so, to improve usability of the SRV resolving feature, default timeout is now 5 seconds. The feature can be disabled or timeout extended to 30 seconds by changing settings.
````ini
resolvesrvrecords=fast # Use 'false', 'fast' (5s timeout), or 'true' (30s timeout).
````
The idea behind this change is to have this working in most case without changing default settings. If SRV resolving does not work, the timeout is short enough to be not too annoying.

**Comment by ORelio** [comment]
This issue should now be properly resolved in #253. Feel free to reopen if that's not the case :)
